src/components/MovieCard/MovieCard.jsx

```diff
@@ -1,5 +1,45 @@
+import { React } from 'react';
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ movie }) => (
+  <div className="card" data-cy="Movie">
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img
+          data-cy="MovieImage"
+          src={movie.imgUrl}
+          alt={movie.title}
+        />
+      </figure>
+    </div>
+
+    <div className="card-content">
+      <div className="media">
+        <div className="media-left">
+          <figure className="image is-48x48">
+            <img src="images/imdb-logo.jpeg" alt={movie.imdbId} />
+          </figure>
+        </div>
+
+        <div className="media-content">
+          <p className="title is-8" data-cy="MovieTitle">
+            {movie.title}
+          </p>
+        </div>
+      </div>
+
+      <div className="content">
+        <p data-cy="MovieDescription">
+          {movie.description}
+        </p>
+
+        <a
+          href="https://www.imdb.com/title/tt1375666"
+          data-cy="MovieLink"
+        >
+          {movie.imdbId}
+        </a>
```

Comment:
must be href={movie.imdbId}, not strange text
  
![image](https://github.com/mate-academy/react_movies-list-js/assets/109717931/eb9ec55a-0d9a-445d-8180-bf928f5600ec)